src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailPendingTasksAlert.jsx

```diff
@@ -1,34 +1,40 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import { getConfig } from '@edx/frontend-platform';
 import { Hyperlink, Alert } from '@edx/paragon';
 import { WarningFilled } from '@edx/paragon/icons';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 
-export default function BulkEmailPendingTasksAlert() {
+export default function BulkEmailPendingTasksAlert(props) {
+  const { courseId } = props;
+
   return (
-    <>
-      <Alert variant="warning" icon={WarningFilled}>
-        <FormattedMessage
-          id="bulk.email.pending.tasks.description.one"
-          defaultMessage="To view all pending tasks, including email, visit&nbsp;"
-        />
-        <Hyperlink
-          destination={`${getConfig().LMS_BASE_URL}/courses/${window.location.pathname.split('/')[2]}/instructor#view-course-info`}
-          target="_blank"
-          isInline
-          showLaunchIcon={false}
-        >
-          <FormattedMessage
-            id="bulk.email.pending.tasks.link"
-            defaultMessage="Course Info"
-          />
-        </Hyperlink>
```

Comment:
These formatting changes are not in the original patch.  Mind reverting them?

Reply:
It was because the original patch was base on node v18, which wasn't mergeed to pam at the time i created it, now it shouldn't be the cased given node 18 is merged and I rebeased